src/main/java/com/unstoppabledomains/config/client/Client.java

```diff
@@ -19,9 +19,23 @@ public static String getVersion() {
 
     private static String initClientVersion() {
```

Comment:
Can we make this method public and add tests?

Reply:
I would leave this private since it's not something that should be called externally. But we can add a test for `getVersion()` to verify if it loads correctly.